src/pages/index.vue

```diff
@@ -1,5 +1,150 @@
-<script setup></script>
+<script setup lang="ts">
+import DcGameCard from "~/components/molecules/DcGameCard.vue";
+import { Game } from "~/types";
+
+const gameList: Game[] = [
+  {
+    id: 1,
+    name: "Lawn Mowing Simulator",
+    platforms: [
+      {
+        id: 1,
+        name: "Windows",
+      },
+      {
+        id: 2,
+        name: "Steam",
+      },
+    ],
+    price: 28.99,
+    img: "https://assets.nuuvem.com/image/upload/t_banner_big/v1/products/610d7e28a3f8b176b24e51fd/banners/kcazjj7yyt8kfdqy0dlv.jpg",
+  },
+  {
+    id: 2,
+    name: "Lawn Mowing Simulator",
+    platforms: [
+      {
+        id: 3,
+        name: "Windows",
+      },
+    ],
+    price: 28.99,
+    img: "https://assets.nuuvem.com/image/upload/t_banner_big/v1/products/610d7e28a3f8b176b24e51fd/banners/kcazjj7yyt8kfdqy0dlv.jpg",
+    promotion: 0.2,
+  },
+  {
+    id: 3,
+    name: "Lawn Mowing Simulator",
+    platforms: [
+      {
+        id: 1,
+        name: "Windows",
+      },
+      {
+        id: 2,
+        name: "Steam",
+      },
+    ],
+    price: 28.99,
+    img: "https://assets.nuuvem.com/image/upload/t_banner_big/v1/products/610d7e28a3f8b176b24e51fd/banners/kcazjj7yyt8kfdqy0dlv.jpg",
+  },
+  {
+    id: 4,
+    name: "Lawn Mowing Simulator",
+    platforms: [
+      {
+        id: 1,
+        name: "Windows",
+      },
+    ],
+    price: 28.99,
+    img: "https://assets.nuuvem.com/image/upload/t_banner_big/v1/products/610d7e28a3f8b176b24e51fd/banners/kcazjj7yyt8kfdqy0dlv.jpg",
+    promotion: 0.7,
+  },
+  {
+    id: 5,
+    name: "Lawn Mowing Simulator",
+    platforms: [
+      {
+        id: 1,
+        name: "Windows",
+      },
+    ],
+    price: 28.99,
+    img: "https://assets.nuuvem.com/image/upload/t_banner_big/v1/products/610d7e28a3f8b176b24e51fd/banners/kcazjj7yyt8kfdqy0dlv.jpg",
+    promotion: 0.7,
+  },
+  {
+    id: 6,
+    name: "Lawn Mowing Simulator",
+    platforms: [
+      {
+        id: 1,
+        name: "Windows",
+      },
+    ],
+    price: 28.99,
+    img: "https://assets.nuuvem.com/image/upload/t_banner_big/v1/products/610d7e28a3f8b176b24e51fd/banners/kcazjj7yyt8kfdqy0dlv.jpg",
+    promotion: 0.7,
+  },
+  {
+    id: 7,
+    name: "Lawn Mowing Simulator",
+    platforms: [
+      {
+        id: 1,
+        name: "Windows",
+      },
+    ],
+    price: 28.99,
+    img: "https://assets.nuuvem.com/image/upload/t_banner_big/v1/products/610d7e28a3f8b176b24e51fd/banners/kcazjj7yyt8kfdqy0dlv.jpg",
+    promotion: 0.7,
+  },
+];
+</script>
 
 <template>
-  <QPage padding> Hello World </QPage>
+  <QPage padding class="row">
+    <div class="home-page__games-grid q-mx-auto q-py-xl row items-center">
+      <div
+        v-for="(game, index) in gameList"
+        :key="game.id"
+        class="col-12 q-pa-md home-page__game-card"
+      >
+        <DcGameCard v-bind="game"> </DcGameCard>
```

Comment:
Why are you using this particular bind?
Going back to check `DcGameCard` component, I could see that we are binding the entire `Game` object as a property. Let's say on the near future we want to add more properties to this DcGameCard (which we might do) that would require a refactory on the component and it's tests.

It is always an option to have a property bound to an entity `:game="game"`

Then we would have something like this:
`const props = defineProps({
  game: Object as PropType<Game>,
});`

The properties on DcGameCard are making a reference to the game not the card. So promotions for instance and img, are properties of the Game and so on.

Reply:
Changed this into an `game: Game` property